README.md

```diff
@@ -43,7 +43,7 @@ What **FuseML** is **NOT**:
 
 ## Inception and Direction
 
-FuseML originated as a fork of our sister open source project [Carrier](https://github.com/SUSE/carrier), a lightweight open source PaaS built on top of Kubernetes, then has been gradually transformed and infused with the MLOps concepts that make it the AI/ML orchestration tool that it is today.
+FuseML originated as a fork of our sister open source project [Fuseml](https://github.com/SUSE/carrier), a lightweight open source PaaS built on top of Kubernetes, then has been gradually transformed and infused with the MLOps concepts that make it the AI/ML orchestration tool that it is today.
```

Comment:
How can FuseML be its own sister project ? :smile:

Reply:
Sorry, I wasn't paying attention